### ISSUE TITLE
Show progress of docker image pull

### ISSUE THREAD

**Issue by apoorvam** [leopardslab/dunner#137]
## Current behaviour

Currently, when a docker image is being pulled from registry, it displays a message that says

`Pulling image: 'golang'`

But it displays no progress of this image pull. This makes user wonder if its really pulling image, especially when network is slow or image is too big. 

## Expected behaviour

Dunner should show progress of docker image pull. 

Something like below that docker shows:

```
ab1fc7e4bf91: Already exists
35fba333ff52: Already exists
f0cb1fa13079: Already exists
3d1dd648b5ad: Already exists
49f7a0920ce1: Pull complete
1d199f738c5f: Pull complete
669e5ed443bf: Pull complete
d8dd16c377d0: Pull complete
Digest: sha256:c9a51b7f2f3774f39058bb436ed62d66f415f32b9386e5c35c3fb4854722ed84
Status: Downloaded newer image for node:10.15.0
```

Or atleast a progress bar indicating how much percentage is complete.

**Comment by agentmilindu** [comment]
Let's have a simple loading kinda thing for the moment.

**Comment by ayushjn20** [comment]
Closed with reference to #147
